src/shared/hibernate-util.c

```diff
@@ -146,19 +146,14 @@ static int read_resume_config(dev_t *ret_devno, uint64_t *ret_offset) {
                 return log_debug_errno(r, "Failed to parse /sys/power/resume devno '%s': %m", devno_str);
 
         r = read_one_line_file("/sys/power/resume_offset", &offset_str);
```

Comment:
please mention explicitly in the git commit message that this raises baseline to 4.17 for the hibernation logic.

And please, can you add this to README in another line like this:

```
                     ≥ 4.17 for hibernation support
```

or so?

it's entirely fine if this is going to be dropped/updated very soon now, but there's no shame in havibg some intermediary line there, for the moment, simply because people backport things and such. i'd rather have too much info on this, than too little